src/shared/lwcDevServerUtils.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2024, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { Workspace } from '@lwc/lwc-dev-server';
+import { Config } from '@salesforce/core';
+import configMeta, { ConfigVars } from '../configMeta.js';
+
+export const LOCAL_DEV_SERVER_DEFAULT_PORT = 8081;
+export const LOCAL_DEV_SERVER_DEFAULT_WORKSPACE = Workspace.SfCli;
+
+export class LwcDevServerUtils {
```

Comment:
FYI, I plan on consolidating `lwcDevServerUtils.ts` and `identityUtils.ts` into `configUtils.ts` since the code in both of these are around fetching or writing different config values. But as far as this PR is concerned, we can leave this unchanged.

Reply:
yeah I realized that we are wasting at least next tick since we are loading the same config twice.